department-of-veterans-affairs/caseflow
# Code cleanup to prep for AMA hearing schedule

Connects #7981 

# Description
This PR refactors the Assign Hearings page to use the queue appeal serializer. This will set us up to easily add AMA appeals as well.

# To test
1) Go to assign hearings page
1) Confirm you can still assign veterans
